src/mito2/src/sst/index/indexer/abort.rs

```diff
@@ -21,7 +21,6 @@ impl Indexer {
         self.do_abort_inverted_index().await;
         self.do_abort_fulltext_index().await;
         self.do_abort_bloom_filter().await;
-        self.do_prune_intm_sst_dir().await;
```

Comment:
So we don't prune sst dir in abort now?

Reply:
The problem is that if we only abort this while we have issues writing the SST/index. It returns an error and thus doesn't have a FileHandle to purge. That's why we clean manually here.

https://github.com/GreptimeTeam/greptimedb/blob/7cf47ccf5430321786204ac8eccec74623a84def/src/mito2/src/sst/parquet/writer.rs#L212-L232